src/cross.rs

```diff
@@ -48,13 +51,37 @@ impl Default for StorableMsg {
     }
 }
 
+#[derive(PartialEq, Eq, Clone, Debug, Default, Serialize, Deserialize)]
+pub struct CrossMsg {
+    pub msg: StorableMsg,
+    pub wrapped: bool,
+}
+
 #[derive(PartialEq, Eq)]
 pub enum IPCMsgType {
     Unknown = 0,
     BottomUp,
     TopDown,
 }
 
+impl CrossMsg {
+    pub fn send<BS, RT>(self, rt: &mut RT, rto: Address) -> Result<RawBytes, ActorError>
```

Comment:
Why not borrow?  (I suspect why but just to double-check)
```suggestion
    pub fn send<BS, RT>(&self, rt: &mut RT, rto: Address) -> Result<RawBytes, ActorError>
```

Reply:
This way the method consumes the message, and avoids cloning. Do you think it's better to do the other way?

Reply:
No, if the message is not needed elsewhere this is the right way :) Thanks for the clarification. I just didn't want to dive into the code and it was easier to double-check with you (apologies for the laziness, but I am all over the place with the Spacenet launch and didn't want to hold this PR longer) :pray: